ODC/latest.json

```diff
@@ -1,3 +1,3 @@
 {
-  "fileName": "assessment_centre_locations_2020_08_20.csv"
+	"fileName": "assessment_centre_locations_2020_08_20.csv"
```

Comment:
This file also appears to be indented with single spaces which suggests that the linting configuration doesn't have full coverage

Reply:
Yes, tabs appear like single spaces on the github review UI, but they are indeed tabs. The linting configuration for checking tabs does cover this file and others.

Reply:
Annoyingly they don't *always* appear like single spaces - for example in this review thread it appears like 8 spaces : P